mam/api/api.c

```diff
@@ -262,45 +263,68 @@ retcode_t mam_api_bundle_write_packet(
       return RC_OOM;
     }
 
+    if (is_last_packet) {
+      ctx->ord = -ctx->ord;
```

Comment:
I think negative is only on the tag and not on the actual format ?

Reply:
But then someone can fake a "last packet" bundle?